views/stylesheet.css

```diff
@@ -12,7 +12,8 @@
 .display {
```

Comment:
since one of the specs were to have all of the font in 'Roboto', i think you could do this instead of setting it to roboto in different elements 

``` * { font-family: 'Roboto', sans-serif; } ```